### PR TITLE
Add JS Bin Swag (#184)

### LINKED ISSUE #184: Add JS Bin swag
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : [JS Bin](https://jsbin.com/)

Q. Where have you seen the announcement (URL)?  
A. : https://jsbin.com/help/stickers/

Q. What kind of swag (clothing, stickers, ...)?  
A. : stickers

Q. Have you received swag from this?  
A. : No

Q. How long did it take?  
A. : N/A

Q. Does it ship worldwide?  
A. : Yes

Note:
> There’s no cost to you, but we will be restricting the send outs to bi-monthly or quaterly depending on demand.

Requirements:
- Have a JS Bin account (has Github Sign-up)

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by bamanczak**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Added JS Bin Swag from Issue #184

Closes #184

### CODE REVIEW

**Comment by vikaspotluri123** [comment]
closes #184, depends on #159 being merged in first

**Comment by bamanczak** [comment]
@vikaspotluri123 #159 is already merged, can we merge this one? :)

File: data.json [hunk 1]
```diff
@@ -98,6 +98,15 @@
     "dateAdded": "2018-10-12T13:25:10.000Z",
     "tags": ["clothing", "hacktoberfest", "stickers"]
   },
+  {
+    "name": "JS Bin",
+    "difficulty": "easy",
+    "description": "Fill out a Google form with your name and address and get a sticker. There’s no cost to you, but they will be restricting the send outs to bi-monthly or quaterly depending on demand.",
+    "reference": "https://jsbin.com/help/stickers/",
+    "image": "https://help.jsbin.com/images/stickers.jpg",
+    "dateAdded": "2018-01-31T15:43:00.000Z",
```
Comment (vikaspotluri123) [inline data.json]:
```suggestion
    "dateAdded": "2018-11-28T16:42:00.000Z",
```

dateAdded is based on issue creation date 🙂

Comment (bamanczak) [reply data.json]: Thanks for the suggestion. To be onest I struggled with defining this one.

Comment (vikaspotluri123) [reply data.json]: No problem! Please let us know if you have any questions 😊

File: data.json [hunk 2]
```diff
@@ -98,6 +98,15 @@
     "dateAdded": "2018-10-12T13:25:10.000Z",
     "tags": ["clothing", "hacktoberfest", "stickers"]
   },
+  {
+    "name": "JS Bin",
+    "difficulty": "easy",
+    "description": "Fill out a Google form with your name and address and get a sticker. There’s no cost to you, but they will be restricting the send outs to bi-monthly or quaterly depending on demand.",
+    "reference": "https://jsbin.com/help/stickers/",
+    "image": "https://help.jsbin.com/images/stickers.jpg",
```
Comment (aslafy-z) [inline data.json]: Let's change this image for:


2019-01-30-150059_349x383_scrot


```suggestion
    "image": "https://pbs.twimg.com/media/CccQEDjUUAE9wRO.jpg",
```

**Comment by aslafy-z** [comment]
Thanks @bamanczak !

**Pull request merged**